src/WebJobs.Extensions.Twilio/Config/TwilioSMSConfiguration.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.Azure.WebJobs.Extensions.Twilio
+{
+    /// <summary>
+    /// Defines the configuration options for the Twilio binding.
+    /// </summary>
+    public class TwilioSmsConfiguration
```

Comment:
Sendgrid probably isn't the best one to follow. The biggest issue is that it doesn't allow each function to set its own Account details and instead relies on the environment variables exclusively.

Once you have it set up to check the Attribute properties first, then look at the Config settings, then fallback to environment, it's a bit easier to do it all in one class. That allows you access to the registered NameResolver as well. 

---
In reply to: [75050079](https://github.com/Azure/azure-webjobs-sdk-extensions/pull/124#discussion_r75050079) [](ancestors = 75050079)